### DISCUSSION TITLE
Open for ideas

### DISCUSSION THREAD

**Discussion by noahschoenthal** [TheVerux/Create-Branch-Name#4] [Ideas]
Hey guys, if you have any ideas write them down.

**Comment by raaph** [comment]
What about adding the possibility to connect to an Azure Devops (or also GitHub) and the possibilty to choose a feature (or issue) from there, which then creates the branch name automatically from the feature (issue, bug, or whatever). If you choose a bug it will have `bug/` instead of `feature/` in the path name

**Reply by noahschoenthal** [reply]
Thanks @raaph for the great idea. I'll consider this.
